sdk/src/utils/serde.rs

```diff
@@ -78,3 +78,50 @@ pub mod option_prefix_hex_vec {
             .transpose()
     }
 }
+
+#[cfg(feature = "client")]
+pub mod bip44 {
+    use crypto::keys::bip44::Bip44;
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Default, Serialize, Deserialize)]
+    #[serde(default, rename_all = "camelCase", remote = "Bip44")]
+    pub struct Bip44Def {
```

Comment:
Can this not just be a module like the others?

Reply:
no